### PR TITLE
More Travis updates

### PR DESCRIPTION
**Pull request by julia326**
- Added vaxrank integration test; also changed PyPi user hammerlab -> openvax.
- Running pylint only on pyensembl files.
- Explicitly adding pip cache directory to Travis config.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -57,14 +81,17 @@ script:
   - pyensembl install --release 87 --species mouse
   - echo "Installed Ensembl 87 for mice" && df -h
   # run tests
-  - nosetests test --verbose --with-coverage --cover-package=pyensembl
+  # -a '!skip' lets us skip certain tests on travis (relevant for vaxrank)
+  # this command runs both mhctools and vaxrank tests. Sometimes they don't print output for a long
```
Comment (tavinathanson) [inline .travis.yml]: Fix comment :)

File: .travis.yml [hunk 2]
```diff
@@ -20,27 +20,51 @@ before_install:
...
-    - pandoc
+      # Needed for NetMHC
+      - tcsh
+      # install pandoc for use with pypandoc for converting the README
+      # from markdown to RST
+      - pandoc
+env:
+  global:
+    # MHC_BUNDLE_PASS
+    - secure: "B2bLOYhDD1mSH6nBmUVoyoVzO8CsZ/qeduoQ/gq1UqPXyBa5cj8x1hYeYEXMVGH04dq6p5FHKIZoZOOmgtKoemje5enSV8IL2lttASqNvzsiyvy6I4C3r5Ve2H9HMLneDxkarfVVVhG2ZJyO7Y/vGN8STZWX6N3FJfCJ1B5kV5Q="
```
Comment (tavinathanson) [inline .travis.yml]: Just confirming that this was also generated using a new `travis encrypt` command?

Comment (julia326) [reply .travis.yml]: yep it was

File: .travis.yml [hunk 3]
```diff
@@ -20,27 +20,51 @@ before_install:
...
+      numpy nose pandas theano pandoc
   - source activate test-environment
   - pip install pypandoc
   - pip install -r requirements.txt
   - pip install .
   - pip install coveralls
+  - pip install --upgrade -r $PWD/vaxrank/requirements.txt
+before_script:
+  # needed for MHCflurry predictor test
+  - mhcflurry-downloads fetch
```
Comment (tavinathanson) [inline .travis.yml]: I think you can remove all `mhcflury` specific stuffs since `vaxrank` doesn't test it?

File: .travis.yml [hunk 4]
```diff
@@ -20,27 +20,51 @@ before_install:
...
+
+  # install MHC predictors
+  - git clone https://mhcbundle:$MHC_BUNDLE_PASS@github.com/openvax/netmhc-bundle.git
+  - export NETMHC_BUNDLE_HOME=$PWD/netmhc-bundle
+  - mkdir tmp
+  - export NETMHC_BUNDLE_TMPDIR=$PWD/tmp
+  - export PATH=$PATH:$NETMHC_BUNDLE_HOME/bin
+
+  # needed for MHCflurry predictor test
+  - export KERAS_BACKEND=theano
```
Comment (tavinathanson) [inline .travis.yml]: Can remove this too

File: .travis.yml [hunk 5]
```diff
@@ -20,27 +20,51 @@ before_install:
...
   # of sequence dictionaries
-  directories: /home/travis/.cache/pyensembl/
+  directories:
+      - /home/travis/.cache/pyensembl/
+      - /home/travis/.cache/pip
 install:
   - >
       conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION
-      numpy nose pandas
+      numpy nose pandas theano pandoc
```
Comment (tavinathanson) [inline .travis.yml]: And remove `theano` here?

File: .travis.yml (hunk 4 above)
Comment (julia326) [reply .travis.yml]: done

File: .travis.yml (hunk 5 above)
Comment (julia326) [reply .travis.yml]: done

File: .travis.yml (hunk 3 above)
Comment (julia326) [reply .travis.yml]: done

**Pull request merged**